src/main/kotlin/com/vk/admstorm/configuration/phpunit/RemotePhpUnitRerunFailedTestsAction.kt

```diff
@@ -61,7 +61,7 @@ open class RemotePhpUnitRerunFailedTestsAction(container: ComponentContainer, pr
             do {
                 do {
                     if (!iterator.hasNext()) {
-                        return ContainerUtil.flatten(result.values)
```

Comment:
Oh! What's broken?

Reply:
JB revoke some functions and add it implicit